python/src/scipp/compat/mantid.py

```diff
@@ -853,6 +853,16 @@ def validate_dim_and_get_mantid_string(unit_dim):
         return known_units[user_k]
 
 
+def var_to_workspace_2d(variable, dim, instrument_file=None):
+    x_vals = np.array(variable.coords[dim].values)
```

Comment:
Variables to not have coords? Do you mean a data array?

Reply:
Yeah, this was written in my first 2 weeks of the project and I've just not updated the name